common/commands/config.go

```diff
@@ -41,6 +41,13 @@ const (
 	WebLogin    AuthenticationMethod = "Web Login"
 )
 
+const (
+	// Indicates that the config command uses OIDC authentication.
+	ConfigOidcCommandName = "config_OIDC"
+	// Default config command name.
+	ConfigCommandName = "config"
```

Comment:
config_OIDC --> config_oidc
Let's check if the above two consts can be made private.